invenio_drafts_resources/services/records/service.py

```diff
@@ -61,6 +64,29 @@ def draft_cls(self):
         """Factory for creating a record class."""
         return self.config.draft_cls
 
+    @property
+    def draft_indexer(self):
```

Comment:
minor: would it be worth to abstract this creation and have a `_indexer` factory method that receives the parameter? I think we would not gain much... maybe code dedup?

Reply:
that could very well come in handy, i can have a look at it on monday!